### PR TITLE
Replacing Topology with Application and fixing grammar mistakes in status text

### PR DESCRIPTION
**Pull request by sureshms**

### CODE REVIEW

**Comment by harshach** [comment]
+1

File: webservice/src/main/resources/app/scripts/components/Sidebar.jsx [hunk 1]
```diff
@@ -204,8 +204,7 @@ export default class Sidebar extends Component {
             : "fa fa-angle-double-left"}></i>
         </a>
         <Modal ref="leaveEditable" onKeyPress={this.handleKeyPress.bind(this)} data-title="Confirm Box" dialogClassName="confirm-box" data-resolve={this.confirmLeave.bind(this, true)}>
-          {< p > Are you sure want to navigate away from this page
-            ? </p>}
+          {< p > Are you sure want to navigate away from this page? </p>}
```
Comment (hmcl) [inline webservice/src/main/resources/app/scripts/components/Sidebar.jsx]: Are you sure **you** want to navigate ...

File: webservice/src/main/resources/app/scripts/utils/Constants.js [hunk 2]
```diff
@@ -29,12 +29,12 @@ const PieChartColor = ["#006ea0", "#77b0bd", "#b7cfdb", "#9dd1e9"];
...
 const deployStatusText = {
-  DEPLOYING_TOPOLOGY : 'Deploying Topology',
-  TOPOLOGY_STATE_EXTRA_JARS_SETUP : 'Topology State Extra Jars Setup',
-  TOPOLOGY_STATE_CLUSTER_ARTIFACTS_SETUP : 'Topology State Cluster Artifacts Setup',
-  TOPOLOGY_STATE_DEPLOYED : 'Topology State Deployed',
-  TOPOLOGY_STATE_SUSPENDED : 'Topology State Suspended',
-  TOPOLOGY_STATE_DEPLOYMENT_FAILED : 'Topology State Deployment Failed'
+  DEPLOYING_TOPOLOGY : 'Deploying Application',
+  TOPOLOGY_STATE_EXTRA_JARS_SETUP : 'Building Application Jars',
+  TOPOLOGY_STATE_CLUSTER_ARTIFACTS_SETUP : 'Setting Up Applcation Artifacts',
```
Comment (hmcl) [inline webservice/src/main/resources/app/scripts/utils/Constants.js]: Application

**Comment by sureshms** [comment]
Let's commit this and address the typos in a subsequent change

Get Outlook for iOS<https://aka.ms/o0ukef>



On Thu, May 25, 2017 at 5:26 PM -0700, "hmcl" <notifications@github.com<mailto:notifications@github.com>> wrote:


@hmcl commented on this pull request.

Two minor typos. Otherwise +1

________________________________

In webservice/src/main/resources/app/scripts/components/Sidebar.jsx<https://github.com/hortonworks/streamline/pull/662#discussion_r118613201>:

> @@ -204,8 +204,7 @@ export default class Sidebar extends Component {
             : "fa fa-angle-double-left"}></i>
         </a>
         <Modal ref="leaveEditable" onKeyPress={this.handleKeyPress.bind(this)} data-title="Confirm Box" dialogClassName="confirm-box" data-resolve={this.confirmLeave.bind(this, true)}>
-          {< p > Are you sure want to navigate away from this page
-            ? </p>}
+          {< p > Are you sure want to navigate away from this page? </p>}


Are you sure you want to navigate ...

________________________________

In webservice/src/main/resources/app/scripts/utils/Constants.js<https://github.com/hortonworks/streamline/pull/662#discussion_r118613569>:

> @@ -29,12 +29,12 @@ const PieChartColor = ["#006ea0", "#77b0bd", "#b7cfdb", "#9dd1e9"];
 let deleteNodeIdArr = [];

 const deployStatusText = {
-  DEPLOYING_TOPOLOGY : 'Deploying Topology',
-  TOPOLOGY_STATE_EXTRA_JARS_SETUP : 'Topology State Extra Jars Setup',
-  TOPOLOGY_STATE_CLUSTER_ARTIFACTS_SETUP : 'Topology State Cluster Artifacts Setup',
-  TOPOLOGY_STATE_DEPLOYED : 'Topology State Deployed',
-  TOPOLOGY_STATE_SUSPENDED : 'Topology State Suspended',
-  TOPOLOGY_STATE_DEPLOYMENT_FAILED : 'Topology State Deployment Failed'
+  DEPLOYING_TOPOLOGY : 'Deploying Application',
+  TOPOLOGY_STATE_EXTRA_JARS_SETUP : 'Building Application Jars',
+  TOPOLOGY_STATE_CLUSTER_ARTIFACTS_SETUP : 'Setting Up Applcation Artifacts',


Application

**Pull request merged**